### ISSUE TITLE
Xbox One Controller not found by driver

### ISSUE THREAD

**Issue by Heyl0** [360Controller/360Controller#1279]
### Type of Controller
Xbox One

### OS Version
13.2.1 (22D68)

### Driver Version
0.16.11_Notarized

### Connection Method
Wired

### Device Name and Info
(If you don't know this information, please refer to the README for how to find it. Enter N/A if not applicable)
>Device Name: Controller
<br/>Product ID: 0x02ea
<br/>Vendor ID: 0x045e  (Microsoft Corporation)

### Response of Controller
Replace this text with information on how the physical controller reacts. Does the guide button light up? Does it vibrate? Etc.

### Response in Preference Pane
Driver "No devices found"
### Detailed Explaination of Issue
Xbox symbol glows
USB Device registers in Hardware Information
Inputs not interpreted

**Comment by iFloneUEFN** [comment]
1. Whats your macbook model
2. Did you disable SIP? It usually does that when your SIP is enabled.

**Comment by Heyl0** [comment]
1. MacBook Air 2019
2.  Yes

**Comment by iFloneUEFN** [comment]
That's odd. Did you allow the driver in Privacy and Security in the settings app?

**Comment by Heyl0** [comment]
yes I did that's why I am really confused why the driver isn't recognising the device that is connected

**Comment by Heyl0** [comment]
I think so, I can double check if you tell me which options to double check I may be able to either fix it or provide screenshots

**Comment by iFloneUEFN** [comment]
Check [this thread](https://github.com/360Controller/360Controller/issues/1139) its for MX, but theres all the steps explained.
